src/helpers/copyContract.js

```diff
@@ -0,0 +1,55 @@
+const RECEIPT_CHECK_TIMEBOX = 100
+const DEPLOYMENT_TIMEOUT = 60000
```

Comment:
Could we use the value of the timeout flag for this?

Reply:
sure